### ISSUE TITLE
Titles Customized via Filters are not Changed in the Admin Preview 

### ISSUE THREAD

**Issue by black-eye** [sybrew/the-seo-framework#383]
Hi, here is the thing:

In my shop I needed to automatically add a brand name before the name of the product, so I used this code:

```php
function bleye_add_brand_to_woo_product( $title ) {

    global $post;
    if ( is_singular( 'product' ) ) {
        $brand_list = wp_get_post_terms($post->ID, 'pwb-brand', array("fields" => "names") );
        if ( ! empty($brand_list) ) {
            $title = $brand_list[0] . ' ' . $title;
        }
    }
    return $title;
}
add_filter( 'the_seo_framework_title_from_generation', 'bleye_add_brand_to_woo_product');
```

It worked quite well on the frontend, brand name (in this case Terranova) was inserted as expected. But in the backend the placeholder for the title still shows the original text taken from the page title (see the screenshot).

![obrazek](https://user-images.githubusercontent.com/5634870/48673141-a7c4ea80-eb3e-11e8-82bf-4b110593eb13.png)

Not a big deal, but for editors it could be quite confusing, that they don't see the real title (and may try to correct it by hand). Is there a way I can update this placeholder too?

**Comment by black-eye** [comment]
Sorry, I wasn't be able to format the code correctly, but I§m sure, you've go the point ;)

**Comment by sybrew** [comment]
I literally faced the same problem a few hours ago. I'm messing about with a new site, testing the capabilities of the filters.

I'm forwarding this bug to issue #88, and I'll mark it as a bug, too.

See my comment there, which explains why this is happening: https://github.com/sybrew/the-seo-framework/issues/88#issuecomment-439679127

As it's a backend-only issue, I'm not going to rush fixing it. I expect to ship a fix with v3.3.0
